xyz.ljoonal.neospeeps.yml

```diff
@@ -0,0 +1,40 @@
+app-id: xyz.ljoonal.neospeeps
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: neos_peeps
+finish-args:
+  # Needs to talk to the network:
+  - --share=network
+  # Wayland access
+  - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
It should, internally just using [egui](https://github.com/emilk/egui) which afaik should support it. Additionally just tested that it at least worksOnMyMachineTM, so bug reports welcome if it doesn't work ^^'